src/colorpicker.cpp

```diff
@@ -102,20 +123,11 @@ void ColorPicker::paintEvent(QPaintEvent *event)
 void ColorPicker::emitColorChanged()
 {
     QImage image = mPixmap.toImage();
-    QColor color = QColor(image.pixel(GRAB_RADIUS, GRAB_RADIUS));
+    QColor color = image.pixelColor(GRAB_RADIUS * mScaleFactor,
+                                    GRAB_RADIUS * mScaleFactor); // introduced in Qt 5.6
```

Comment:
Can you remove this comment? It's not useful.

Reply:
Sure, it was somewhat for reference as "if you are OK to have 5.6 as minimum version"